lib/data/managers/firestore_manager.dart

```diff
@@ -148,4 +166,19 @@ class FireStoreManager {
     }
     return listOfFeaturedArticles;
   }
-}
+
+  Future<ArticleEntity> getArticleById(String articlePathReference) async {
+    ArticleEntity articleEntity;
+    if (articlePathReference != null) {
+      await Firestore.instance.document(FLAME_LINK_CONTENT+"/"+articlePathReference).get().then(
+        (featuredArticlesSnapshot) async {
```

Comment:
Actually is fetching only featured articles because you cannot share an article that is not featured article, but i dont have any problem to change it to fetchedArticleSnapshot